src/tbp/monty/frameworks/environments/embodied_data.py

```diff
@@ -715,6 +735,10 @@ def handle_successful_jump(self):
 
         else:
             self.get_good_view("view_finder")
```

Comment:
How about `get_good_view_with_patch_refinement`? It's a long name, but multi-step to me sounds like it could refer to multiple steps taken with the same sensor (as is partly done already with move forward).

Reply:
Yeah good point, I like it!

Reply:
Updated.